docs/_posts/murat-gunay/2021-09-09-bert_token_classifier_ner_btc_en.md

```diff
@@ -52,7 +51,7 @@ ner_converter = NerConverter()\
 pipeline =  Pipeline(stages=[documentAssembler, tokenizer, tokenClassifier, ner_converter])
 
 model = pipeline.fit(spark.createDataFrame(pd.DataFrame({'text': ['']})))
-test_sentences = ["""Wengers big mistakes is not being ruthless enough with bad players.""", """my dream FUUUUUULHAAAAAAM !!!.."""]
+test_sentences = ["""Wengers big mistakes is not being ruthless enough with bad players.""", """my dream Fulham F.C. !!!.."""]
```

Comment:
Can you remove second example and use this one?

![image](https://user-images.githubusercontent.com/36634572/144835637-5e84b6af-026f-4351-b6ca-f749b087ee9f.png)